lib/blocksim.py

```diff
@@ -127,6 +127,67 @@ def change_state(self, x):
     def derivative(self, e):
         return 0
 
+class DiscreteTF(Block):
+    
+    def __init__(self, name, input_name, output_name, dt, numerator, denominator):
+        """
+        Simulates a discrete transfer function.        
+        The TF must be of the form: \frac{b_N z^{-N} + ... + b_0 }{a_M z^{-M} + ... + a_0 }.
```

Comment:
LaTeX notation will not be rendered automatically - please make this plain text.